### ISSUE TITLE
NTR: Anti-GBM-antibody positivity

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5801]
**Preferred term label:**
Anti-GBM-antibody positivity

**Synonyms**
Anti-glomerular basement membrane-antibody

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react to the glomerular basement membrane.

Comment: Anti-GBM-antibodies are found in Goodpasture syndrome. Circulating antibodies are directed against the collagen of the part of the kidney known as the glomerular basement membrane (GBM), resulting in acute or rapidly progressive glomerulonephritis. Antibodies also attack the collagen of the air sacs of the lung (alveoli) resulting in bleeding of the lung (pulmonary hemorrhage).

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Goodpasture Syndrome OMIM:233450
ORPHA:375   (Disorder)   Anti-glomerular basement membrane disease

**Your nano-attribution (ORCID)**
lmu: chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
done
